content/composer/available-actions.md

```diff
@@ -6,42 +6,42 @@ draft: false
 weight: 300
 
 ---
+## Actions Bar
+On the top right you can find additional actions to perform on the current used blueprint.
 
-On the top right you can find additional actions to perform on the current opened blueprint.
-
-# Save
+### Save Blueprint
 Save will trigger two actions:
 
 -  Saving the displayed blueprint.
 -  Running a validation check on the blueprint.yaml.
 
-# Download
+### Download Blueprint 
 Downloads the last saved blueprint - packages and downloads an archive of the blueprint including all folders with resources & plugins as a tar file.
 
-# Validate
+### Validate Blueprint
 Validates the displayed blueprint source code to ensure logical concepts are kept.
 
-# Logout
-Will route the user back to login page.
-
-# Blueprint Settings
+### Blueprint Settings
 The settings button opens up a menu of operations to be performed:
 
 ![Blueprint Composer Blueprint Settings]({{< img "ui/composer/blueprint-settings.png" >}})
 
-## Add New
+#### Add New Blueprint 
 Creates a new empty blueprint canvas.
 
-## Rename
+#### Rename a Blueprint
 Allows changing the displayed blueprint name.
 
-## Import
+#### Import a Blueprint 
 Opens an existing blueprint in the composer canvas, the blueprint can either be a local file or a url accessible by the composer.
 
-## Delete
+#### Delete a Blueprint 
 Removes the blueprint both from the display and the composer saved data and cannot be undone.
 
-# Browsing Blueprints
+### Switching Between Composed Blueprints
 To browse a user’s blueprints, click on the arrow next to the blueprint name on the top left and choose one of the available blueprints to work on. Blueprints displayed would be the ones the user created, imported, and saved.
 
-![Blueprint Composer Browsing Blueprints]({{< img "ui/composer/browsing-blueprints.png" >}})
\ No newline at end of file
+### Logout (From Composer) 
+Logout will destroy current user session and route the user back to login page allowing a new user to login.
```

Comment:
Destroy current user session sounds like everything is about to be deleted. I think it's better to write:
"Logout ends the current user session and routes the user..."